src/resources/js/aggregator-fields.js

```diff
@@ -105,6 +105,10 @@ tribe_aggregator.fields = {
 				obj.$.form.attr( 'data-origin', $( this ).val() );
 				obj.reset_preview();
 
+				// reset all bumpdowns
+				$( '.tribe-bumpdown-active' ).removeClass( 'tribe-bumpdown-active' );
```

Comment:
Could we use the selectors that are saved on the `selectors.dropdown` variable? (or update it)